src/open_inwoner/templates/pages/product/detail.html

```diff
@@ -34,6 +36,10 @@ <h1 class="h1" id="title">
     <p class="p">{{ object.summary }}</p>
     {{ object.content|ckeditor_content|safe }}
 
+    {% if question_set or files or conditions or locations or product_links or published_related_products or contacts %}
+        <hr class="divider">
```

Comment:
If would probably have done this differently, by not adding a real HR divider, but making it a CSS only **border** that would appear if the class "has" something inside it or not - I was hoping Django would NOT show Divs if they do not contain anything. If the templates just simply do not render Divs that do not contain anything, you can do something in CSS like this:

`.product-info:has(.faq),
.product-info:has(.map),
.product-info:has(.link-list),
.product-info:has(.notification) {
  padding-top: var(--spacing-giant);
  margin-top: var(--spacing-giant);
  border-top: 2px solid var(--color-gray-light);
}`

But this does not work in Firefox...!